common/types/shared.go

```
@@ -7842,3 +8083,9 @@ type RespondCrossClusterTasksCompletedResponse struct {
 type StickyWorkerUnavailableError struct {
 	Message string `json:"message,required"`
 }
+
+// SerializeRequest Serialize an arbitrary request for logging
+func SerializeRequest(request interface{}) string {
+	res, _ := json.Marshal(request)
```

It's not good to ignore the error here.